client/components/LoginPage.tsx

```diff
@@ -1,44 +1,93 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { 
+  TextField, 
+  Box, 
+  Typography, 
+  InputAdornment,
+  IconButton,
+  Button
+} from "@mui/material";
+import {
+  Visibility,
+  VisibilityOff
+} from "@mui/icons-material";
+
 import { useLoginMutation } from "../features/api";
 
 const LoginPage = () => {
   const [login, { isLoading, isError, data }] = useLoginMutation();
 
+  const navigate = useNavigate();
+
+
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+  };
 
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     login({ email, password });
+    navigate("/");
   };
 
   return (
-      <form onSubmit={handleSubmit}>
-        <h1>Login</h1>
-        {isError && <p>Oops, there was an error logging in. Try again?</p>}
-
-        <label htmlFor="email">Email</label>
-        <input
-          type="email"
-          id="email"
-          name="email"
-          onChange={(e) => setEmail(e.target.value)}
-          value={email}
-        />
-
-        <label htmlFor="password">Password</label>
-        <input
-          type="password"
-          id="password"
-          name="password"
-          onChange={(e) => setPassword(e.target.value)}
-          value={password}
-        />
-
-        <button disabled={isLoading} type="submit">
-          Login
-        </button>
-      </form>
+    <Box 
+      component="form"
+      onSubmit={handleSubmit}
+      autoComplete="off"
+    >
+      <Typography component="h1">Login</Typography>
```

Comment:
This is probably an h2, not an h1 - the h1 would be the name of the app, and there should generally only be one h1 on the page